app/assets/stylesheets/components/_messages.scss

```diff
@@ -0,0 +1,26 @@
+@mixin flash($color) {
+  $text-color: if(lightness($color) < 65%, lighten($color, 35%), darken($color, 35%));
+
+  background: $color;
+  border-radius: $base-border-radius;
+  color: $text-color;
+  display: block;
+  font-weight: $base-font-weight;
+  margin-bottom: $base-spacing;
+  margin-top: 1em;
+  padding: $small-spacing;
+  text-shadow: 0 1px 0 rgba(#000, 0.07);
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.